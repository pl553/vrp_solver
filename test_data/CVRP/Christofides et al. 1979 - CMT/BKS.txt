CMT01	524.61
CMT03	826.14
CMT06	555.43
CMT07	909.68
CMT12	819.56
CMT14	866.37
CMT02	835.26
CMT04	1028.42
CMT05	1291.29
CMT08	865.94
CMT09	1162.55
CMT11	1042.11
CMT13	1541.14
CMT10	1395.85
