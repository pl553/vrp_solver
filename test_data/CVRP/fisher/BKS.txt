F-n045-k4	724.00
F-n072-k4	237.00
F-n135-k7	1162.00
