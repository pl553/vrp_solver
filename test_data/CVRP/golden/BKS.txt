Golden_03	11036.22
Golden_05	6460.98
Golden_04	13592.88
Golden_06	8404.26
Golden_13	857.19
Golden_14	1080.55
Golden_17	707.76
Golden_18	995.13
Golden_01	5623.47
Golden_02	8404.61
Golden_07	10102.68
Golden_08	11635.34
Golden_09	579.71
Golden_10	736.26
Golden_11	912.84
Golden_12	1102.69
Golden_15	1337.92
Golden_16	1612.50
Golden_19	1365.60
Golden_20	1818.32

